### ISSUE TITLE
APK's permissions, ".obb not found" message

### ISSUE THREAD

**Issue by gitthisissue** [saber-nyan/ddlc-android#17]
So I tried installing your APK to my phone, and this app required these permissions:

modify or delete SD card
read the data from SD card

_com.android.vending.CHECK_LICENSE_
_full network access_
_view network connections_
_view WLAN connections_

control vibration
prevent phone from sleeping

---

I don't understand why these 4:
_com.android.vending.CHECK_LICENSE_
_full network access_
_view network connections_
_view WLAN connections_
is required. Can't you make this app standalone (play without internet)?

Also, after I install the app and open it, it closed automatically
and showed this notification:

"Doki Doki Literature Club!
Looking for resource to download"

Good thing is I don't connect to internet. What are you trying to download?

**Comment by saber-nyan** [comment]
> I don't understand why these 4:
> com.android.vending.CHECK_LICENSE
> full network access
> view network connections
> view WLAN connections

These are the permissions that are requested by any application built on the Ren'Py engine. Basically, I can build an application without these permissions, but I can't guarantee its correct operation: I will have to modify RAPT files (build system for Android).

> Can't you make this app standalone (play without internet)?

This application does not require the Internet.

> Also, after I install the app and open it, it closed automatically
> and showed this notification:
> 
> "Doki Doki Literature Club!
> Looking for resource to download"

You forgot to put the `.obb` file into `/sdcard/Android/obb/ru.saber_nyan.ddlc/`. It can be downloaded in the Releases section.

> What are you trying to download?

Nothing! Honestly.

**Comment by gitthisissue** [comment]
> requested by any application built on the Ren'Py engine

Makes sense. If possible, would you try to disable them? Android geeks won't download any app which require internet permission to protect their device.

> You forgot to put the .obb file into /sdcard/Android/obb/ru.saber_nyan.ddlc/

Yep, my bad. But hey, how about adding an appropriate message instead of "Looking for resource to download" notification?

e.g. Show dialogue when .ogg not found
"Oops!
You forgot to place our OBB file to sdcard path. Please read our wiki. The app is closing now.
[ OK ]"

FYI: There's many device which doesn't have SD card slot. Try scanning these path too.
/storage/sdcard
/storage/sdcard0
/storage/sdcard1
/storage/emulated/0
/storage/emulated/1

All above path is correct which leads you to sdcard's root path.



> What are you trying to download?
> Nothing! Honestly.

Confirmed by glancing all your source codes :)

**Comment by saber-nyan** [comment]
> If possible, would you try to disable them?

I'll try to do it in the next alpha.

> how about adding an appropriate message instead of "Looking for resource to download" notification?

Eh, this is again part of RAPT, which I have to rewrite...
Maybe I'll do it.

> Try scanning these path too.

Part of RAPT!
Btw, `/sdcard/` is a symlink to internal memory, unless it has been changed by the user or developer of a firmware...
